gov-notify-stub/gov-notify-spec.yaml

```diff
@@ -0,0 +1,305 @@
+  ## ---------------------
+  responses:
+    400:
+      description: Bad Request
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            PAYLOAD_VALIDATION:
+              $ref: "#/components/examples/PayloadValidationError"
+            MALFORMED_DIGEST:
+              $ref: "#/components/examples/MalformedRequestDigestError"
+            MALFORMED_TOKEN:
+              $ref: "#/components/examples/MalformedRequestTokenError"
+    401:
+      description: Unauthorized
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            BAD_TOKEN:
+              $ref: "#/components/examples/BadTokenError"
+            BAD_SDK_ID:
+              $ref: "#/components/examples/BadSdkIdError"
+            BAD_PUBLIC_KEY:
+              $ref: "#/components/examples/BadPublicKeyError"
+            MESSAGE_SIGNING:
+              $ref: "#/components/examples/MessageSigningError"
+    403:
+      description: Auth Error
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            SCHEME_NOT_FOUND:
+              $ref: "#/components/examples/SchemeNotFoundError"
+            SCHEME_VALIDATION_REQUIREMENT_NOT_FOUND:
+              $ref: "#/components/examples/SchemeValidationRequirementNotFoundError"
+    429:
+      description: Too Many Requests Error
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            TOO_MANY_REQUESTS:
+              $ref: "#/components/examples/TooManyRequestsError"
+    500:
+      description: Internal Server Error
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            SERVER_ERROR:
+              $ref: "#/components/examples/UnexpectedServerError"
+    503:
+      description: Service Unavailable
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorResponse"
+          examples:
+            TEMPORARILY_UNAVAILABLE:
+              $ref: "#/components/examples/TemporarilyUnavailableError"
+
+  ## ---------------------
+  ## Security Schemes
+  ## ---------------------
+  securitySchemes:
+    X-Yoti-Auth-Digest:
+      type: apiKey
+      name: X-Yoti-Auth-Digest
+      in: header
+      description: "Signed request performed by back-end SDKs that includes a `X-Yoti-Auth-Digest`"
+    X-Yoti-Auth-Token:
+      type: apiKey
+      name: X-Yoti-Auth-Token
+      in: header
+      description: "`X-Yoti-Auth-Token` header containing the `client_session_token` value obtained via the `Session` creation endpoint."
+
+  ## ---------------------
+  ## Example Definitions
+  ## ---------------------
+  examples:
```

Comment:
are these examples all relevant to the GovNotify stub?

Reply:
Not required, Removed the unwanted definitions and schemas.